### DISCUSSION TITLE
Does OpenTelemetry supports seq logs using python.

### DISCUSSION THREAD

**Discussion by lallaw8809** [datalust/seq-tickets#1926] [General]
I would like to send the logs to seq using OpenTelemetry.  
But I could not find any tutorial to do it.  

Is it possible to send the logs to seq using OpenTelemetry python?

**Comment by nblumhardt** [comment]
Hi @lallaw8809, thanks for dropping by.

The OTLP logging support in Python seems like it's only just starting to come together, but there's an example at:

https://github.com/open-telemetry/opentelemetry-python/blob/main/docs/examples/logs/example.py

which should have everything you need.

You'll have to set the `endpoint` argument to `OTLPLogExporter` to the HTTPS address of your Seq instance, e.g.:

```
OTLPLogExporter(endpoint="https://your-seq-server")
```

You'll also need to remove any tracing/metrics configuration from the example.

Let me know if this helps!

Best regards,
Nick

**Reply by lallaw8809** [reply]
Sorry for the late reply. I was on vacation.
It works, I am able to send the logs to seq.

**Reply by nblumhardt** [reply]
Fantastic, thanks for letting us know 👍
